playground/nextjs/app/app-dir/sign-in/page.tsx

```diff
@@ -1,5 +1,11 @@
 import { SignIn } from '@clerk/nextjs';
 
 export default function Page() {
-  return <SignIn path='/app-dir/sign-in' signUpUrl='/app-dir/sign-up' />
+  return (
+    <SignIn
+      path='/app-dir/sign-in'
```

Comment:
Just checking - do we need to define `path` here? I think it's getting picked up by the env, correct?

Reply:
If we don't pass it we get an error. Also we want different paths for app router and pages so having it in the env will not work.